File: engine/model/src/main/scala/net/server/channel/packet/newyear/CardHasBeenSentPacket.scala
package net.server.channel.packet.newyear

class CardHasBeenSentPacket(private var _reqMode: Byte, private var _slot: Short, private var _itemId: Int, private var _receiver: String, private var _message: String) extends BaseNewYearCardPacket(_reqMode = _reqMode) {
  def slot: Short = _slot

  def itemId: Int = _itemId

  def receiver: String = _receiver

  def message: String = _message
}

File: engine/engine-model/src/main/java/net/server/channel/packet/newyear/CardHasBeenSentPacket.java
package net.server.channel.packet.newyear;

public class CardHasBeenSentPacket extends BaseNewYearCardPacket {
   private final Short slot;

   private final Integer itemId;

   private final String receiver;

   private final String message;

   public CardHasBeenSentPacket(Byte reqMode, Short slot, Integer itemId, String receiver, String message) {
      super(reqMode);
      this.slot = slot;
      this.itemId = itemId;
      this.receiver = receiver;
      this.message = message;
   }

   public Short slot() {
      return slot;
   }

   public Integer itemId() {
      return itemId;
   }

   public String receiver() {
      return receiver;
   }

   public String message() {
      return message;
   }
}
